projects/sandbox/pyproject.toml

```diff
@@ -262,3 +262,5 @@ log_file = "${base.basedir}/log/sensitive_volume.log"
 max_far = 1000
 sigma = 0.1
 verbose = "${base.verbose}"
+gate_paths = {"H1" = "/home/ethan.marx/public_html/forks/BBHNet/projects/sandbox/vizapp/vizapp/vetoes/H1-O3_GATES_1238166018-31197600.txt", "L1" = "/home/ethan.marx/public_html/forks/BBHNet/projects/sandbox/vizapp/vizapp/vetoes/L1-O3_GATES_1238166018-31197600.txt"}
```

Comment:
how big are these files? Are they worth adding to version control?

Reply:
They're versioned already in the vizapp project

Reply:
These files live in a git.ligo.org repo so maybe we can have the script download them and store them via an environment variable or something

Reply:
@alecgunny I added ability to apply vetoes to foreground events, and removed the paths to vetoes as arguments of the function in favor of pointing directly to the versioned control versions.